Cartfile

```diff
@@ -7,3 +7,4 @@ github "DaveWoodCom/XCGLogger"              ~> 7.0
 github "cezheng/Fuzi"                       ~> 3.0
 github "SwiftyJSON/SwiftyJSON"              ~> 5.0
 github "google/EarlGrey"                    ~> 1.15
+github "mozilla/application-services"       "v0.32.3"
```

Comment:
it was remove from Cartfile but not from Cartfile.resolved - it has to stay as it provides bookmarks storage